conf/modules.config

```diff
@@ -305,9 +305,17 @@ process {
         ext.args = params.bowtie2_mode ? params.bowtie2_mode : params.ancient_dna ? '--very-sensitive -N 1' : ''
         ext.prefix = { "${meta.id}.assembly" }
         publishDir = [
-            path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
-            mode: params.publish_dir_mode,
-            pattern: "*.log"
+            [
+                path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
+                mode: params.publish_dir_mode,
+                pattern: "*.log"
+            ],
+            [
+                path: { "${params.outdir}/Assembly/${assembly_meta.assembler}/QC/${assembly_meta.id}" },
```

Comment:
Maybe `Assembly/${assembly_meta.assembler}/bams/${assembly_meta.id}` or similar? Or even `AssemblyMaps/${assembly_meta.assembler}/${assembly_meta.id}/`? Not sure that bams/bais fit into the context of 'QC'!

Reply:
I  think QC would be okay because the alignment can be used to quality check the assemblies, but agree that keeping it broader (like bam/maps) might work better because these BAM files have other uses as well.

Reply:
Exactly, I'm keeping them alongside where the bowtie2 log file is saved... I don't want to rock the boat too much v😬

Reply:
If you agree we can reconfigure the output in the future, then could you give a ✅!